formats/compact-refract.md

```diff
@@ -0,0 +1,96 @@
+# Compact Refract
+
+Compact Refract is a serialization format Refract for the purpose of removing a
+lot of the object keys that are repeated throughout the full serialization of
+Refract, as seen in the Refract specification. It also allows for expressing
+structures in a tuple, which resembles other formats like XML or Lisp.
+
+## Dependencies
+
+- [Refract Base
+Specification](https://github.com/refractproject/refract-spec/blob/master/refract-spec.md)
+  - Only Include
+    - Element Pointer
+
+## Data Structures
+
+### Compact Element (array)
+
+The Compact Element is a tuple where each item has a specific meaning. The
+first item is the element name, the second is the meta attribute section, the
+third is the attribute section, and the fourth is the content section.
+
+#### Members
+
+- (string, required) - Name of the element
+- (Meta Attributes, required) - Meta attributes of the element instance.
+- (object, required) - Attributes of the element instance
+- (enum, required) - Element content with any of the following types
+  - (null)
+  - (string)
+  - (number)
+  - (boolean)
+  - (array)
+  - (object)
+  - (Compact Element)
+  - (array[Compact Element])
+
+## Meta Attributes
+
+The Meta Attributes are similar to those defined by the base specification,
+though the only difference is that Link Elements are of a different structure.
+
+### Members
+
+- `id` - Unique Identifier, MUST be unique throughout the document
+- `ref` (Element Pointer) - Pointer to referenced element or type
+- `classes` (array[string]) - Array of classifications for given element
+- `title` (string) - Human-readable title of element
+- `description` (string) - Human-readable description of element
+- `links` (array[Link Element]) - Meta links for a given element
+
+## Link Element (array)
```

Comment:
Is this required here? We don't define `Element Pointer` or `string` here either since it's in the base spec, I assume.